grimoire_elk_surveyqq/enriched/surveyqq.py

```diff
@@ -146,7 +143,7 @@ def get_sh_identity(self, item_answer, identity_field=None):
         identity['username'] = user_answer[0]["text"]
         identity['email'] = user_answer[1]["text"]
         identity['name'] = None
-        return identity
+        return [identity]
```

Comment:
Why adding bracket?

Reply:
I try to return a list containing a dict.
new identity like  [{'username':'xxxx','email': 'xx@xx','name':None }]

Reply:
> 

I try to return a list containing a dict.
new identity like [{'username':'xxxx','email': 'xx@xx','name':None }]

Reply:
Thanks for your explaination!